bigtable/beam/workload-generator/src/test/java/bigtable/WorkloadGeneratorTest.java

```diff
@@ -0,0 +1,191 @@
+import com.google.bigtable.repackaged.com.google.monitoring.v3.ListTimeSeriesRequest;
+import com.google.bigtable.repackaged.com.google.monitoring.v3.ProjectName;
+import com.google.bigtable.repackaged.com.google.monitoring.v3.TimeInterval;
+import com.google.bigtable.repackaged.com.google.monitoring.v3.TimeSeries;
+import com.google.bigtable.repackaged.com.google.protobuf.util.Timestamps;
+import com.google.cloud.bigtable.beam.CloudBigtableTableConfiguration;
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Date;
+import java.util.UUID;
+import org.apache.beam.runners.dataflow.DataflowClient;
+import org.apache.beam.runners.dataflow.DataflowPipelineJob;
+import org.apache.beam.runners.dataflow.DataflowRunner;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.Create;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Admin;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WorkloadGeneratorTest {
+
+  private static final String INSTANCE_ENV = "BIGTABLE_TESTING_INSTANCE";
+  private static final String TABLE_ID =
+      "mobile-time-series-" + UUID.randomUUID().toString().substring(0, 20);
+  private static final String COLUMN_FAMILY_NAME = "stats_summary";
+
+  private static String projectId;
+  private static String instanceId;
+  private static final String REGION_ID = "us-central1";
+
+  private ByteArrayOutputStream bout;
+
+  private static String requireEnv(String varName) {
+    String value = System.getenv(varName);
+    assertNotNull(
+        String.format("Environment variable '%s' is required to perform these tests.", varName),
+        value);
+    return value;
+  }
+
+  @BeforeClass
+  public static void beforeClass() {
+    projectId = requireEnv("GOOGLE_CLOUD_PROJECT");
+    instanceId = requireEnv(INSTANCE_ENV);
+    try (Connection connection = BigtableConfiguration.connect(projectId, instanceId)) {
+      Admin admin = connection.getAdmin();
+      HTableDescriptor descriptor = new HTableDescriptor(TableName.valueOf(TABLE_ID));
+      descriptor.addFamily(new HColumnDescriptor(COLUMN_FAMILY_NAME));
+      admin.createTable(descriptor);
+    } catch (Exception e) {
+      System.out.println("Error during beforeClass: \n" + e.toString());
+    }
+  }
+
+  @Before
+  public void setupStream() {
+    // bout = new ByteArrayOutputStream();
```

Comment:
do we need this?

Reply:
yes, I uncommented for debugging, will re add